README.md

```diff
@@ -42,22 +42,22 @@ go install github.com/caddyserver/xcaddy/cmd/xcaddy@latest
 Example Caddyfile (replace `user` and `pass` accordingly):
 ```
 {
-  servers {
-    protocols h1 h2 h3
-  }
+	order forward_proxy before file_server
+	servers :443 {
+		protocols h1 h2 h3
+	}
```

Comment:
Not needed, already the default, https://caddyserver.com/docs/caddyfile/options#protocols